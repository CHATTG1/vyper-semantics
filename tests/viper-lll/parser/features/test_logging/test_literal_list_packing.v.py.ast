%pgm(
  %event(Bar, %eparam(_value, %listT(%num, 4), false)), ,
  %fdecl(%@public, foo, , %void,
    %log(Bar, %list(1 2 3 4)))
)
