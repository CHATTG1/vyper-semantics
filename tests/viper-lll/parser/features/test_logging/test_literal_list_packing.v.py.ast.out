$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 3264763256 ) , $%seq ( $pass , $assert ( $iszero ( $callvalue ) ) , $%seq ( $%seq ( $mstore ( 320 , 1 ) , $mstore ( 352 , 2 ) , $mstore ( 384 , 3 ) , $mstore ( 416 , 4 ) ) , $log1 ( 320 , 128 , 109018203172587760336500144510619617288757461242172758304564404389609541436238 ) ) , $stop ) ) ) , 0 ) ) )
